src/test/java/org/vafer/jdependency/ClazzpathUnitTestCase.java

```diff
@@ -19,24 +19,40 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 import static org.junit.Assert.assertEquals;
-
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.vafer.jdependency.Clazz.parseClassFileName;
 
 public class ClazzpathUnitTestCase {
 
     private static Path resourcePath( String filename ) {
-        return Paths.get(filename);
+        ClassLoader classLoader = ClazzpathUnitTestCase.class.getClassLoader();
+        URL resource = classLoader.getResource(filename);
+        if (resource == null) {
+            return null;
+        }
+        return Paths.get(resource.getFile());
```

Comment:
Looks good is using the unit test classloader strictly required?

Reply:
If it works it works. I just didn't find it necessary yet. (Which IDE are you using?)

Reply:
I use IntelliJ on Linux